pkg/virt-controller/watch/vm/vm.go

```diff
@@ -517,6 +517,14 @@ func (c *Controller) handleDataVolumes(vm *virtv1.VirtualMachine) (bool, error)
 		} else if curDataVolume.Status.Phase != cdiv1.Succeeded &&
 			curDataVolume.Status.Phase != cdiv1.WaitForFirstConsumer &&
 			curDataVolume.Status.Phase != cdiv1.PendingPopulation {
+
+			if curDataVolume.Status.Phase == cdiv1.Pending {
+				err := storagetypes.HasDataVolumeExceededQuota(vm.Namespace, vm.Spec.Template.Spec.Volumes, c.dataVolumeStore)
```

Comment:
nit: inline with the error checking

Reply:
when the `err` assignment line is long, I tend not to inline the `if`. Most of this file aligns with this convention. wdyt?